server/db/knexfile.js

```diff
@@ -8,7 +8,12 @@ dotenv.config({
 
 module.exports = {
   client: 'pg',
-  connection: process.env.DATABASE_URL,
+  connection: {
+    connectionString: process.env.DATABASE_URL,
+    ssl: {
```

Comment:
It seems it will always try to connect using SSL, but this will cause an error if the server does not support SSL connections.

Reply:
Proposed fix:

```js
function buildSSLConfig() {
  if (process.env.REJECT_UNAUTHORIZED_SSL_CERTIFICATE) {
    return {
      rejectUnauthorized: false,
    };
  }

  return false;
}

module.exports = {
  client: 'pg',
  connection: {
    connectionString: process.env.DATABASE_URL,
    ssl: buildSSLConfig(),
  },
  migrations: {
    tableName: 'migration',
    directory: path.join(__dirname, 'migrations'),
  },
  seeds: {
    directory: path.join(__dirname, 'seeds'),
  },
  wrapIdentifier: (value, origImpl) => origImpl(_.snakeCase(value)),
};
```

I will create a PR and also add documentation for environment variable within the `.env.sample` file. The default equals to true, so connections that don't require a SSL connection will be functional without any changes.